common/src/java/com/zimbra/common/soap/AdminConstants.java

```diff
@@ -1303,6 +1304,13 @@ public final class AdminConstants {
     public static final String A_VOLUME_USE_IN_FREQ_ACCESS = "useInFrequentAccess";
     public static final String A_VOLUME_USE_IN_FREQ_ACCESS_THRESHOLD = "useInFrequentAccessThreshold";
```

Comment:
If these are new fields, s/InFrequent/Infrequent/.  "Infrequent" is not two words. I'm pretty sure it was correctly one word in the design documents I've read.

If it was already here, then I guess we keep it for all eternity like the "Referer" header misspelling in HTTP.